en/can/README.md

```diff
@@ -0,0 +1,88 @@
+  It has not yet seen significant adoption.
+
+:::note
+Both DroneCAN and Cyphal originate from an earlier project named UAVCAN.
+In 2022 the project split into two: the original version of UAVCAN (UAVCAN v0) was renamed to DroneCan, and UAVCAN v1 was renamed Cyphal.
+:::
+
+:::warning
+PX4 does not support other CAN software protocols for drones such as KDECAN (at time of writing).
+:::
+
+## Wiring
+
+The wiring for CAN networks is the same for both DroneCAN and Cyphal (in fact, for all CAN networks).
+
+Devices are connected in a chain in any order.
+At either end of the chain, a 120Ω termination resistor should be connected between the two data lines.
+Flight controllers and some GNSS modules have built in termination resistors for convenience, thus should be placed at opposite ends of the chain.
+Otherwise, you can use a termination resistor such as [this one from Zubax Electronics](https://shop.zubax.com/products/uavcan-micro-termination-plug?variant=6007985111069), or solder one yourself if you have access to a JST-GH crimper.
+
+The following diagram shows an example of a CAN bus connecting a flight controller to 4 CAN ESCs and a GNSS.
+
+![CAN Wiring](../../assets/can/uavcan_wiring.png)
+
+The diagram does not show any power wiring.
+Refer to your manufacturer instructions to confirm whether components require separate power or can be powered from the CAN bus itself.
+
+For more information, see [Cyphal/CAN device interconnection](https://kb.zubax.com/pages/viewpage.action?pageId=2195476) (kb.zubax.com).
+While the article is written with the Cyphal protocol in mind, it applies equally to DroneCAN hardware and any other CAN setup.
+
+### Connectors
+
+Pixhawk standard compatible CAN devices use 4 pin JST-GH connectors for CAN.
+Two connectors are used for input and output when wiring in a chain (except for flight controllers and some GNSS devices with builtin termination, which only have a single JST-GH connector).
+
+Other (non-Pixhawk compatible) devices may not use these connectors.
+However, as long as the device firmware supports DroneCAN or Cyphal, it can be used.
+
+### Redundancy
+
+DroneCAN and Cyphal support using a second (redundant) CAN interface.
+This is completely optional but increases the robustness of the connection.
+All Pixhawk flight controllers come with 2 CAN interfaces; if your peripherals support 2 CAN interfaces as well, it is recommended to wire both up for increased safety.
+
+## Firmware
+
+Some CAN peripherals come with custom firmware written by the manufacturer.
+However, in addition to firmware for flight controllers, PX4 can be run as peripheral firmware itself on supported devices.
+See the [DroneCAN Support Table](../dronecan/support.md) for more information.
+
+## Support and Configuration
+
+[DroneCAN Setup and Configuration](../dronecan/README.md)
+
+[DroneCAN Support Table](../dronecan/support.md)
```

Comment:
This is empty. It is supposed to be the set of supported devices right? I'd be happy for that to be in the DroneCan docs OR it could be in its own docs. If it is in its own doc as implied, then support is a bad name because it implies "how do I get support for DroneCAN problems" not "what devices are supported"

So this would be DroneCan Devices or similar.

It should be something like